client-js/app/elements/labrad-grapher-dataset.html

```diff
@@ -18,15 +30,17 @@
   Polymer({
     is: 'labrad-grapher-dataset',
     properties: {
-      items: {
-        type: Array,
+      path: {
+        type: String,
         notify: true,
+      },
+      dataset: {
+        type: String,
+        notify: true
       }
     },
-    ready: function() {
-      this.items = [
-        'A single dataset'
-      ];
+    back: function(){
+      window.location.href='/grapher/'+this.path.toString().replace(/,/g,"/");
```

Comment:
Actually scratch that, I figured it out and it would involve a change to app.ts function ```loadDataset``` but I wanted to talk to you about what data structures we were sending back and forth for the grapher before changing this. I wanted to limit this pull request to cosmetic changes, not touching the underlying logic.